data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,555 @@
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 60
+	cargo "food (tribute)" 35 2 .05
+	on complete
+		payment 7000 2000
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 4 5
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+mission "Unfettered Tribute 3"
+	name "Hai Tribute to <planet>"
+	job
+	repeat
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 40
+	cargo "food (tribute)" 45 2 .05
+	on complete
+		payment 9000 2400
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 6
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+
+
+phrase "unfettered boarded"
+	word
+		`"This is not a fair fight. It is better to die with my crew standing than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`
+		`"This ship is the property of Unfettered, Hai even, but never monkey! We'd rather die than let you have it."`
+		`"It is truly sad it had to come to this, but we know you will not listen to reason. We have no choice but to destroy our ship."`
+
+mission "Proud Unfettered Boarded"
+	boarding
+	repeat
+	minor
+	to offer
+		"flagship crew" > 120
+		random < "flagship crew" / 8
+	source
+		government "Hai (Unfettered)"
+		category "Heavy Warship"
+	on offer
+		conversation
+			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message.`
+		dialog phrase "unfettered boarded"
+		conversation
+			`It is no bluff... Best get out of here!`
+				flee
+	destination "Darkcloak"
+
+
+
+phrase "unfettered mining payment dialog"
+	word
+		`The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use those asteroids as target practice, you know, but most find it dull and too easy to fight an enemy that does not shoot or react in any other way," one tells you as it hands over your payment of <payment> credits.`
+		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it." One tells you as it hands over your payment of <payment> credits.`
```

Comment:
```suggestion
		`The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships! Too few captains are patient enough to gather asteroids like you are. Most of them try to get the resources from enemy ships, but this is a safer way to do it," one tells you as it hands over your payment of <payment>.`
```